xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1120,7 +1120,7 @@ _io_stats_write_latency_sample(xlator_t *this, ios_sample_t *sample,
         sprintf(group_name, "%d", (int32_t)sample->gid);
     }
 
-    ios_log(this, logfp, "%0.6lf,%s,%s,%0.4lf,%s,%s,%s,%s,%s,%s", epoch_time,
+    ios_log(this, logfp, "%0.6lf,%s,%s,%"PRIu64",%s,%s,%s,%s,%s,%s", epoch_time,
```

Comment:
```suggestion
    ios_log(this, logfp, "%lu.%06lu,%s,%s,%"PRIu64",%s,%s,%s,%s,%s,%s", sample->timestamp.tv_sec, sample->timestamp.tv_nsec / 1000,
```

We could also show the time in nanoseconds and avoid the division.